src/pages/careers/locations/_location-layout.js

```diff
@@ -15,6 +15,16 @@ const Pin = styled.img`
     height: 24px;
     margin-right: 13px;
 `
+
+const StyledBackground = styled(BackgroundImage)`
+    min-width: 100%;
```

Comment:
Is there any difference between `min-width` and `width` here?

Reply:
Reverted this changed in the latest commit. Thanks!